### ISSUE TITLE
Beatmap with storyboards that replace background often look wrong where there is content behind them

### ISSUE THREAD

**Issue by bdach** [ppy/osu#22238]
Opening mostly for tracking as a fix for this was attempted before this was even properly reported (https://github.com/ppy/osu/pull/22035).

The case where this reproduces include:

1. Have screen scaling on, have storyboards fully dimmed. Play beatmap whose storyboard replaces background; the dimmed background that appears when scaling is on will appear behind player.
2. Go to spectator 

### Discussed in https://github.com/ppy/osu/discussions/22211

<div type='discussions-op-text'>

<sup>Originally posted by **Inconsist** January 15, 2023</sup>
The video speaks for itself, is this an issue or my fault? It happend right after I left a lobby and then joined back in to spectate, my dim setting is at 100% and everything worked as intended 5 min. prior to the recording within the same session.

https://user-images.githubusercontent.com/87947227/212521333-83134ece-3aab-409f-90d3-e0497ec58940.mp4

https://user-images.githubusercontent.com/87947227/212521335-64d51365-206e-4867-9f92-cb6e1a969fc9.mp4

</div>

**Comment by Inconsist** [comment]
In case this adds any other info for you, I had screenscaling off during that recording, so it seems to be possible to encounter this even without it.

**Comment by bdach** [comment]
Yeah we're aware thanks. There are many distinct scenarios where this reproduces.

**Comment by Inconsist** [comment]
alright just wanted to make sure :)

**Comment by Inconsist** [comment]
https://osu.ppy.sh/beatmapsets/1544600#osu/3157178 another map where this reproduces

**Comment by cdwcgt** [comment]
I actually think the easiest way is to add a black background behind the storyboard, because storyboarders always assume the background is black

**Comment by frenzibyte** [comment]
That is not adding anything, re-read https://github.com/ppy/osu/issues/22238#issue-1535382713, it's expensive, needs some level of profiling.
